ExpenseTracker/ExpenseTracker/Views/Account/ConfirmEmail.cshtml

```diff
@@ -10,7 +10,7 @@
     <div class="confirmation-container">
         <h2>Email Confirmed</h2>
         <p>Thank you! Your email has been successfully confirmed. You can now proceed to login and start using the app.</p>
-        <a href="/login" class="btn btn-confirm">Go to Login</a>
+        <a href="/account/login" class="btn btn-confirm">Go to Login</a>
```

Comment:
use **asp-action** instead of **href**. In case of conflicts just remove it.

Reply:
asp-action="Login"  done